### PR TITLE
Integration test on a local storage

### PR DESCRIPTION
**Pull request by LukasReschke**
From https://github.com/owncloud/core/pull/25705

### CODE REVIEW

**Comment by LukasReschke** [comment]
LGTM

File: build/integration/features/provisioning-v1.feature [hunk 1]
```diff
@@ -304,19 +305,19 @@ Feature: provisioning
 
 	Scenario: enable an app
 		Given As an "admin"
-		And app "files_external" is disabled
-		When sending "POST" to "/cloud/apps/files_external"
+		And app "user_ldap" is disabled
+		When sending "POST" to "/cloud/apps/user_ldap"
```
Comment (nickvergessen) [inline build/integration/features/provisioning-v1.feature]: Can we use an app that has no php lib deps?

Comment (LukasReschke) [inline build/integration/features/provisioning-v1.feature]: Let me use `testing` then

**Comment by LukasReschke** [comment]
@nickvergessen Done. Please rereview.

**Comment by nickvergessen** [comment]
:+1:

**Comment by LukasReschke** [comment]
stable10 at https://github.com/nextcloud/server/pull/1206

**Pull request merged**